namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-jobs-board-reporting-preprod/00-namespace.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     cloud-platform.justice.gov.uk/business-unit: "HMPPS"
     cloud-platform.justice.gov.uk/slack-channel: "prison-education"
     cloud-platform.justice.gov.uk/application: "Jobs Board Reporting"
-    cloud-platform.justice.gov.uk/owner: "Education Skills Work and Employment: managing.eswe@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/owner: "Education Skills Work and Employment: getreadyforwork@justice.gov.uk"
```

Comment:
Please can this be updated to
`"Education Skills and Work: getreadyforwork@justice.gov.uk"`